docs/asciidoc/faq.adoc

```diff
@@ -206,4 +206,102 @@ processor.publishOn(scheduler1)
          .subscribe();
 ----
 
+== What is a good pattern for contextual logging? (MDC)
+As we saw in section <<context>>, `ThreadLocal` doesn't make much sense in Reactive Streams, where several streams could execute steps on a common thread.
```

Comment:
Change "As we saw" to "As described" (because the reader may not have seen that yet). Change "doesn't" to "does not" (formal documentation does not use contractions).  Change "execute" to "run" (less violent connotation).